firmware/processor.c

```diff
@@ -280,9 +306,12 @@ static const struct video_timing video_modes[PROCESSOR_MODE_COUNT] = {
 		.v_blanking = 17,
 		.v_sync_offset = 1,
 		.v_sync_width = 3,
+
+		.flags = 0x1e,
```

Comment:
Yep, I'll change it to using the defines. I put that in before I had the constants, and forgot to go back and change it.

It is quite likely some of these values should be something else. 0x1e was previously used for everything, which is why I kept it, but the value should really match the mode.